### DISCUSSION TITLE
sha256 for netbeans never matches

### DISCUSSION THREAD

**Discussion by jimgwhit** [apache/netbeans#3673] [General]
The sha256 for netbeans-12.4-bin.zip is:

2b41e310eff2c3d87edbe6390afc72606d7e931f8a1fd3529fe42c282b2f87fac6fe29582d8cb2f778997f1fb0cf0ae011f46d7cca591ad0c0c5dea7c944a708

However when I run:
C:\WINDOWS\system32>certUtil -hashfile C:\Users\Owner\Downloads\netbeans-12.6-bin.zip sha256

I get:  95ea3f8aab954ee44e68ee2f2a8828e46f91fda86efa9312b1186e7736ab8b49

Similar with two previous releases, no match, please help, I checked following their instructions.

**Comment by vieiro** [comment]
Apache NetBeans binary releases (including 12.4) use SHA-512, not SHA-256. You can appreciate that yourself if you compare the length of the two hashes.

**Reply by vieiro** [reply]
Having said that, the SHA-512 of NetBeans 12.4 is not that one you're posting, the correct one is here https://netbeans.apache.org/download/nb124/nb124.html
